integration/containerd/cri/integration-tests.sh

```diff
@@ -98,15 +99,8 @@ ci_cleanup() {
 		sudo cp "$default_containerd_config_backup" "$default_containerd_config"
 	fi
 
-	ID=${ID:-""}
-	if [ "$ID" == ubuntu ]; then
-		if [ -n "${SNAP_CI}" ]; then
-			# restore default configuration
-			sudo cp "${default_kata_config}" "${kata_config}"
-		elif [ -n "${CI}" ] ;then
-			[ -f "${kata_config}" ] && sudo rm "${kata_config}"
-		fi
-	fi
+	[ -f "$kata_config_backup" ] && sudo mv "$kata_config_backup" "$kata_config" || \
+		sudo cp "$default_kata_config" "$kata_config"
```

Comment:
Why is the copy of the default kata config needed here?

This diverges from what was done before, which was dependent on `$SNAP_CI` and `$CI` values.

Reply:
Hmm, good call. I'd actually like to keep the simplified logic suggested here rather than checking `$CI` etc. and just create a backup if a non-default config exists and restore it. However, it would still be better to remove the config in this case.